frontend/src/pages/EmployeeProfile/EmployeeSkillsAndCourses/AddCourseButton/AddCourseDialog/CoursesList/CourseItem/CourseItem.tsx

```diff
@@ -10,17 +10,10 @@ interface IProps {
   isChecked: boolean;
   title: string;
   handleCheckboxChange: (event: BaseSyntheticEvent) => void;
-  courseRef?: (node?: Element | null) => void;
 }
 
-const CourseItem: FC<IProps> = ({
-  checkboxValue,
-  isChecked,
-  title,
-  handleCheckboxChange,
-  courseRef,
-}) => (
-  <StyledItem ref={courseRef}>
+const CourseItem: FC<IProps> = ({ checkboxValue, isChecked, title, handleCheckboxChange }) => (
```

Comment:
Could we change interface name as ICourseItemProps?

Reply:
We dont have one solution for this cases. Somebody wants to create props interfaces inside types.ts, somebody inside component file. In this case i just removed on incoming prop, Before changing this case, we should found one solution. We can discuss it on ower meeting in Tuesday